### DISCUSSION TITLE
How to limit the fields in the nested model

### DISCUSSION THREAD

**Discussion by junqed** [pydantic/pydantic#3931] [Question]
Hello! 

Here's a small question. I have two models and want export the whole Outer object + Inner with the limited number of fields.

```python
from pydantic import BaseModel, Field

class Inner(BaseModel):
    id: int
    name: str
    some_field: str

class Outer(BaseModel):
    id: int
    name: str
    # here a lot of other fields
    inner: Inner = Field(include={'id', 'name'})

print(Outer.parse_obj({'id': 1, 'name': "some-name", "inner": {"id": 1, "name": "some-inner-name", "some_field": "field content"}}).dict())
```
The result is: `{'inner': {'id': 1, 'name': 'some-inner-name'}}`

Adding to every field of the Outer class `include=True` is not what I wanted. Is there any way to solve this issue? I haven't found the answer in the manual. Thanks.

**Comment by suimong** [comment]
You could try to add excluded fields in the `Config` class of `Inner`, something like this:
```
class Inner(BaseModel):
    id: int
    name: str
    some_field: str

    class Config:
        fields = {"some_field": {"exclude": True}}
```

**Reply by junqed** [reply]
But what if I need to use these fields only it's a part of the Outer instance? Inner can be used as a separate instance where all fields are required

**Reply by suimong** [reply]
I guess you could write a wrapper method around the `dict` method for `Outer` and explicitly pass the `exclude={"inner": {"some_field"}}` argument to the original `dict` method?

**Reply by junqed** [reply]
Well, yes, this probably a workaround, but what about `from_orm`?
